compiler-rt/lib/builtins/udivmoddi4.c

```diff
@@ -82,7 +82,7 @@ COMPILER_RT_ABI du_int __udivmoddi4(du_int a, du_int b, du_int *rem) {
         r.s.high = n.s.high & (d.s.high - 1);
         *rem = r.all;
       }
-      return n.s.high >> __builtin_ctz(d.s.high);
```

Comment:
what is the problem with using __builtin?

Reply:
The problem is `__builtin_ctz` has `int` aka `int16_t` argument.